crates/webidl-tests/dictionary.rs

```diff
@@ -56,3 +56,30 @@ fn required() {
 fn correct_casing_in_js() {
     assert_camel_case(PreserveNames::new().weird_field_name(1));
 }
+
+#[wasm_bindgen_test]
+fn roundtrip_set_get() {
+    let mut many_types = ManyTypes::new();
+    many_types.a("a");
+    many_types.n1(1);
+    many_types.n2(2);
+    many_types.n3(3);
+    many_types.n4(4);
+    many_types.n5(5);
+    many_types.n6(6);
+    let mut other_dict = OtherDict::new();
+    other_dict.a(42);
+    many_types.c(&other_dict);
+
+    {
+        use crate::generated::ManyTypesGetters;
+        assert_eq!(many_types.a(), "a");
+        assert_eq!(many_types.n1(), 1);
+        assert_eq!(many_types.n2(), 2);
+        assert_eq!(many_types.n3(), 3);
+        assert_eq!(many_types.n4(), 4);
+        assert_eq!(many_types.n5(), 5);
+        assert_eq!(many_types.n6(), 6);
+        assert_eq!(many_types.c(), other_dict);
+    }
```

Comment:
Unfortunately, due to the signature name clash this is a bit awkward to use right now.

If you move the `use crate::generated::ManyTypesGetters;` to the parent block, this test breaks.